lib/iris/analysis/cartography.py

```diff
@@ -500,8 +501,20 @@ def project(cube, target_proj, nx=None, ny=None):
             covering the globe.
 
     Returns:
-        An instance of :class:`iris.cube.Cube` and a list describing the
-        extent of the projection.
+        An tuple containing an instance of :class:`iris.cube.Cube` and a list
+        describing the extent of the projection.
+
+    Example::
+
+        from iris.analysis.cartography import project
+
+        iris_cs = iris.coord_systems.Stereographic()
+        iris_cs_cube = project(cube, iris_cs)
+
+        cartopy_cs = ccrs.PlateCarree()
+        cartopy_cs_cube = project(cube, cartopy_cs)
```

Comment:
In both these examples I think it'd be clearer to unpack the return value, e.g.:
```python
reprojected_cube, extent = project(cube, cartopy_cs)
```